corehq/apps/geospatial/urls.py

```diff
@@ -23,6 +24,8 @@
         mapbox_routing_status,
         name="mapbox_routing_status"),
     url(r'^settings/$', GeospatialConfigPage.as_view(), name=GeospatialConfigPage.urlname),
+    url(r'^case_grouping_settings/$', ConfigureCaseGroupingView.as_view(),
```

Comment:
Since there's already a `settings/` path, can I suggest that we change this path to `settings/case-grouping`?

Reply:
See [5c38756](https://github.com/dimagi/commcare-hq/pull/33507/commits/5c387564d588c71eed30415fce058496d7f23d19).